test.py

```diff
@@ -14,24 +14,37 @@
 
```

Comment:
Nose does that on purpose, so we just have to make sure our test files don't have the executable bit set.